homeassistant/components/netgear/device_tracker.py

```diff
@@ -153,21 +142,21 @@ def _update_info(self):
 
         Returns boolean if scanning successful.
         """
-        if not self.success_init:
-            return
-
-        _LOGGER.info("Scanning")
+        _LOGGER.debug("Scanning")
 
         results = self.get_attached_devices()
 
+        if _LOGGER.isEnabledFor(logging.DEBUG):
+            print("DEBUG LOGGING")
```

Comment:
When I set the `logger` integration to info level for this platform, this print is done. Am I doing it wrong somehow?

Reply:
It seems our `logger` integration only sets filters on the logging handlers not the loggers. So changing the level in the `logger` integration won't affect the logging level for the loggers.

https://docs.python.org/3/howto/logging.html#logging-flow

Reply:
Oh 🤦‍♂

Reply:
I can keep the guard, so that if we fix the `logger` integration, it will work. What do you think?

Reply:
Yeah that sounds good.